ui/src/screens/Talents/TalentsScreen.tsx

```diff
@@ -65,18 +66,24 @@ const TalentsScreen: React.FC = function () {
       </Helmet>
       <Search placeholder="Search talents" onChange={onSearchChange} />
       <CategorySelector category="talents" />
-      <PostsContainer>
-        {state.talents.map((talent: PostInterface) => (
-          <Post category="talents" active={talent.active} key={talent.id} data={talent} />
-        ))}
-        <PostLoader hasMore={state.more} onLoadMore={handleLoadMore} />
-      </PostsContainer>
+      { state.fetched ? (
+        <PostsContainer>
+          {state.talents.map((talent: PostInterface) => (
+            <Post category="talents" active={talent.active} key={talent.id} data={talent} />
+          ))}
+          <PostLoader hasMore={state.more} onLoadMore={handleLoadMore} />
+        </PostsContainer>
+      ) : (
+          <PostsContainer>
+            <PostSpinner />
+          </PostsContainer>
+        )}
       <DetailsContainer active={active}>
         {state.activeTalent ? (
           <PostDetail data={state.activeTalent} category="talents" />
         ) : (
-          <PostDetailPlaceholder type="talents" />
-        )}
+            <PostDetailPlaceholder type="talents" />
```

Comment:
@jacobtyq Can you double check that your prettier is working? There are spacing issues in this file. If your prettier is setup, then perhaps you did not enable "format on save" in your editor?

Reply:
Ah yes my format on save was unchecked.